src/intro-to-c/enums/enum-tutorial.md

```diff
@@ -0,0 +1,168 @@
+
+``` c
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11};
+```
+
+In this case, the assigned values would look like this:
+
+```
+wheat = 2
+onion = 3
+carrot = 4
+corn = 7
+potatoes = 8
+strawberries = 11
+```
+
+This is in line with the "previous + 1" rule stated earlier. `onion`, `carrot`, and `potatoes` will all follow this rule because they were not explicitly defined.
+
+Values in the same list can have the same values. If we look back at our `game_difficulty` example, we can add a few more titles that have equivalent difficulty.
+
+``` c
+enum game_difficulty {Cookie_Clicker = 1, Gungeon = 6, ARK = 7, Dark_Souls = 10, Super_Meat_Boy = 10, House_Flipper = 1, Bloodborne = 10};
+```
+
+As you can see, Dark Souls, Bloodborne, and Super Meat Boy are all rated at the same level of difficulty, as well as House Flipper and Cookie Clicker. These will not cause any errors, they just happen to hold the same value, as if they were any regular `int` variable.
+
+## Using Enums in Code
+
+When using enums, we must define a variable type. This can be done in one of two ways. I will reuse the `exp_value` example.
+
+```c
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11};
+enum exp_value crops;
+
+// or
+
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} crops;
+```
+
+We can even define multiple variables of type `exp_value` like this:
+
+```c
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} plants;
+enum exp_value crops, dirt_growers;
+
+// or
+
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} crops, plants, dirt_growers;
+```
+
+Now we can assign and use our enumerated values in code. Lets look at something simple.
+
+``` c
+#include <stdio.h>
+
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} crops, plants, dirt_growers;
+
+int main(void)
+{
+	crops = onion;
+	plants = corn;
+	dirt_growers = potatoes;
+	printf("Onion Experience Gain: %d\n", crops);
+	printf("Corn Experience Gain: %d\n", plants);
+	printf("Potatoes Experience Gain: %d\n", dirt_growers);
+	return 0;
+}
+```
+
+Here we assign the value of `onion` to `crops`, which we know is 3. The expected output would be as follows:
+
+```
+Onion Experience Gain: 3
+Corn Experience Gain: 7
+Potatoes Experience Gain: 8
+```
+
+But what if we want to get a bit more tricky with our enums, like this maybe?
+
+``` c
+int x = 8
+crops Mystery_crop = x;
+```
+
+The compiler will warn you that it is expecting a semicolon after `crops` if you try to define an enumerator variable this way without the use of `typedef`.
+
+### Using `typedef` With Enums
+
+We can use the `typedef` keyword to make our enumerated list a bit more versatile. For example, we will be able to create an enum variable, rather than being restricted to the constants in the enumerated list. Let's look at this example.
+
```

Comment:
Like the other suggestion, I may first write the generic version to show what the syntax is.

i.e.:
``` c
typedef <enum_definition> <enum_alias>;

// or

typedef enum <enum_name> {[enumerator1, enumerator2, ...]} <enum_alias>;
```

Reply:
Did that, too. Thank you.